build_library/build_image_util.sh

```diff
@@ -836,9 +836,16 @@ EOF
     sudo setfiles -Dv -r "${root_fs_dir}" "${root_fs_dir}"/etc/selinux/mcs/contexts/files/file_contexts "${root_fs_dir}"/etc
   fi
 
-  # Backup the /etc contents to /usr/share/flatcar/etc to serve as source
-  # for creating missing files
+  # Backup the /etc contents to /usr/share/flatcar/etc to serve as
+  # source for creating missing files. Make sure that the preexisting
+  # /usr/share/flatcar/etc does not have any meaningful (non-empty)
+  # files, so we remove nothing important.
+  if [[ $(sudo find "${root_fs_dir}/usr/share/flatcar/etc" -size -1 2>/dev/null) -gt 0 ]]; then
```

Comment:
I am not sure how that works.
Should we probably add `| wc -l` at the end of find, to get a number of lines?

Reply:
Meh, there actually used to be `| wc -l`. But I updated it and must have removed that by mistake… Thanks for spotting, will fix it.

Reply:
I kicked off another image job run just to make sure that nothing breaks: http://jenkins.infra.kinvolk.io:8080/job/container/job/image/2698/cldsv/

Reply:
Image job passed.